packages/tenant-process/test/db.test.ts

```diff
@@ -65,8 +85,12 @@ describe("database test", () => {
     tenantService = tenantServiceBuilder(postgresDB, readModelService);
   });
 
+  const mockEService = getMockEService();
+  const mockDescriptor = getMockDescriptor();
+
   afterEach(async () => {
     await tenants.deleteMany({});
+    await agreements.deleteMany({});
```

Comment:
Since you also insert eServices in the `readModel`, here you should delete them as well.